src/System.IO.Pipelines.Text.Primitives/ReadableBufferExtensions.cs

```diff
@@ -220,7 +223,7 @@ public static unsafe string GetUtf8String(this ReadableBuffer buffer)
 
                 buffer.CopyTo(destination);
 
-                textSpan = destination.Slice(0, buffer.Length);
+                textSpan = destination.Slice(0, (int) buffer.Length);
```

Comment:
Can you please add a comment here: // We are able to cast because buffer.Length < 128?